app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -68,6 +68,25 @@ class BookmarksTest {
         }
     }
 
+    @Test
+    fun verifyEmptyBookmarksMenuTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openBookmarks {
+            bookmarksListIdlingResource =
```

Comment:
I see this being used in other tests too, but wondering if this can be moved to a robot instead of having it here which makes the tests more difficult to read/follow...